src/moonstone-samples/src/All/All.js

```diff
@@ -135,7 +135,7 @@ module.exports = kind({
 		{from: 'title', to: '$.listpanel.title'}
 	],
 	listTools: [
-		{kind: Panels, pattern: 'activity', hasCloseButton: false, components: [
```

Comment:
Had various discussions with @Djspaceg about this and it seemed better to keep it consistent as it is a full-screen panels-based application.